src/features/Lock/lock.js

```diff
@@ -411,7 +418,7 @@ const Lock = () => {
                         </Button>
                       ) : (
                         <Button
-                          disabled={buttonIsDisabled}
+                          disabled={buttonIsDisabled()}
```

Comment:
wasn't this working without making it a explicit call?

Reply:
It wasn't